styles/hackage/catppuccin.user.css

```diff
@@ -0,0 +1,574 @@
+    .embedded-author-content {
+      border-color: @accent-color;
+    }
+
+    /*
+     * Contents & synopsis
+     */
+
+    #contents-list {
+      background-color: @mantle;
+    }
+
+    #table-of-contents {
+      background-color: @base;
+      border-color: @surface1;
+    }
+
+    #synopsis summary {
+      color: transparent;
+    }
+
+    #synopsis ul,
+    #synopsis ul li.src {
+      background-color: @surface0;
+    }
+
+    /*
+     * Tables
+     */
+
+    table.info {
+      background-color: @base;
+      color: @subtext1;
+      border-color: @surface1;
+    }
+
+    .doc th,
+    .doc td table.fancy,
+    table.fancy tr {
+      border-color: @surface1;
+    }
+
+    .doc th,
+    table.fancy tr.even td {
+      background-color: @mantle;
+    }
+
+    table.fancy td,
+    table.properties td,
+    table.fancy th,
+    table.properties th,
+    table.dataTable.compact.fancy tbody th,
+    table.dataTable.compact.fancy tbody td {
+      background-color: @base;
+    }
+
+    table.fancy,
+    table.fancy tr {
+      border-color: @surface1;
+    }
+
+    div#table_length.dataTables_length,
+    div#table_filter.dataTables_filter {
+      color: @text;
+    }
+
+    div #description table,
+    div #description table tr td {
+      border-color: @surface1;
+    }
+
+    div #description table th {
+      background-color: @mantle;
+    }
+
+    /*
+     * Source links
+     */
+
+    #interface .src .selflink,
+    #interface .src .link {
+      color: @overlay2;
+    }
+
+    #interface span.fixity {
+      color: @overlay1;
+      border-left-color: @overlay1;
+    }
+
+    #interface span.rightedge {
+      border-left-color: @overlay1;
+    }
+
+    #interface p.src .link {
+      background-color: transparent;
+      color: @overlay2;
+      border-color: @overlay1;
+    }
+
+    /* Paginator */
+
+    .paginator span {
+      color: @text;
+    }
+
+    .paginator a {
+      color: @overlay1;
+    }
+
+    .paginator a[href]:link,
+    .paginator a:link:hover {
+      color: @accent-color;
+    }
+
+    .paginator a:visited,
+    .paginator a:visited:hover {
+      color: mix(@text, @accent-color, 75%);
+    }
+
+    .paginator a:link:hover,
+    .paginator a:visited:hover {
+      background-image: none;
+      background-color: @surface1;
+      border-color: transparent;
+    }
+
+    .paginator .current,
+    .paginator .current:hover {
+      background-image: none;
+      background-color: @surface0;
+      color: @accent-color;
+      border-color: transparent;
+    }
+
+    .paginate_button {
+      background-color: @overlay0;
+    }
+
+    /*
+     * Search query
+     */
+
+    #browseTable th {
+      border-color: @surface1;
+    }
+
+    #fatalError {
+      color: @red;
+    }
+
+    input:invalid {
+      border-color: @yellow;
+      box-shadow: 0 0 10px @yellow;
+    }
+
+    input {
+      background-color: @mantle;
+      border-style: solid;
+      border-color: @surface0;
+      border-radius: 4px;
```

Comment:
Do we need to change `border-style` and `border-radius` here?
EDIT: Ah I guess you are completely customizing the UI to make it even more CTP, makes sense IG - exception to the "only change colors" rule?